### ISSUE TITLE
warning: instance variable @enable_for_subprocesses not initialized

### ISSUE THREAD

**Issue by tagliala** [simplecov-ruby/simplecov#926]
Hi,

I would like to submit a PR by myself but I cannot find time at the moment and since I'm not using the subprocess functionality it would be hard to debug

#881 introduced a warning on variable `@enable_for_subprocesses`, which is going to be used even when it is not initialized (`value is nil`).

I don't know if it is correct to use the same or-equal approach as `at_exit` configuration

You can see the warning here: https://travis-ci.org/github/tagliala/coveralls-ruby-reborn/jobs/724960704

Log Snippet
```
............./home/travis/build/tagliala/coveralls-ruby-reborn/spec/coveralls/simple_cov/formatter_spec.rb:15: warning: Passing the keyword argument as the last hash parameter is deprecated
/home/travis/build/tagliala/coveralls-ruby-reborn/vendor/bundle/ruby/2.7.0/gems/simplecov-0.19.0/lib/simplecov/result.rb:28: warning: The called method `initialize' is defined here
......./home/travis/build/tagliala/coveralls-ruby-reborn/vendor/bundle/ruby/2.7.0/gems/simplecov-0.19.0/lib/simplecov/configuration.rb:203: warning: instance variable @enable_for_subprocesses not initialized
```

**Comment by PragTob** [comment]
:wave: 

Hi there,

thanks for the report. Curious. We're supposed to catch warnings like that in CI but maybe in the CI migration something went wrong. Dang.

**Comment by PragTob** [comment]
yeah it's only tested in rspec not in cucumber and the feature was only covered by cukes. Sorry about that :)

**Comment by tagliala** [comment]
No problems, thanks for the heads-up

Apologies for not having submitted a PR, I think you may know if `||=` has been intentionally avoided

**Comment by EdwardIII** [comment]
Hey, thanks for all your work on a brilliant useful tool! Can we upgrade to specific version for now to get rid of the error, or is a fix still in the works?

We see the error in minitest with these gems installed on the project. Happy to try a pre-release test or pitch in with a solution if you can point me in the right direction.

```
$ bundle install
Using rake 13.0.1
Using concurrent-ruby 1.1.7
Using i18n 1.8.5
Using minitest 5.14.2
Using thread_safe 0.3.6
Using tzinfo 1.2.7
Using zeitwerk 2.4.0
Using activesupport 6.0.3.4
Using ast 2.4.1
Using bundler 2.1.2
Using byebug 11.1.3
Using coderay 1.1.3
Using docile 1.3.2
Using method_source 1.0.0
Using parallel 1.19.2
Using parser 2.7.2.0
Using pry 0.13.1
Using pry-byebug 3.9.0
Using rainbow 3.0.0
Using regexp_parser 1.8.1
Using rexml 3.2.4
Using rubocop-ast 0.7.1
Using ruby-progressbar 1.10.1
Using unicode-display_width 1.7.0
Using rubocop 0.92.0
Using simplecov-html 0.12.3
Using simplecov 0.19.0
Bundle complete! 6 Gemfile dependencies, 27 gems now installed.
```

**Comment by PragTob** [comment]
Hey there,

sorry I haven't managed to dedicate a lot of time to OSS lately. First fix didn't go fast as we had lots of fun with the CI being broken.... aiming for a release some time next week depending on how it goes.

Cheers,
Tobi

**Comment by EdwardIII** [comment]
Thanks for the update. Anything I can help with on the CI? I've setup CIs using Jenkins, CodeBuild, gitlab runners etc so might be able to pitch in if that's useful.

**Comment by PragTob** [comment]
@EdwardIII nah but thanks. We just had CI fail randomly on acceptance test related specs which presumably was due to a Chrome upgrade that broke something. See https://github.com/simplecov-ruby/simplecov/issues/925

**Comment by tagliala** [comment]
Thanks, also the feature test to make sure that there are no warnings is a great addition

**Comment by PragTob** [comment]
Yeah I always wanted to have something like it... we are already checking it for the rspec unit test (I wrote essentially a small library for this) but the thing wasn't unit tested. I was never sure how to correctly implement it as a feature test as ideally I'd want to check it in all tests (just like rspec) but I realized that one broad high level test running for most of the paths is better than nothing 😁 

One of the reasons this took so long, because I wanted to have it covered. Sorry and here's to no more such problems!
